src/registrar/templates/domain_dsdata.html

```diff
@@ -104,20 +81,40 @@ <h2 class="margin-top-0">DS Data record {{forloop.counter}}</h2>
     </button>
 
     <button
+          id="save-ds-data"
           type="submit"
           class="usa-button"
       >Save
     </button>
-  </form>
 
-  <form aria-label="form to undo changes to the DS records">
     <button
           type="submit"
-          class="usa-button usa-button--outline btn-cancel"
+          class="usa-button usa-button--outline"
           name="btn-cancel-click"
           aria-label="Reset the data in the DS records to the registry state (undo changes)"
       >Cancel
     </button>
   </form>
+
+  {% if trigger_modal %}
+  <a
+      id="ds-toggle-dnssec-alert"
+      href="#toggle-dnssec-alert"
+      class="usa-button usa-button--outline margin-top-1 display-none"
+      aria-controls="toggle-dnssec-alert"
+      data-open-modal
+      >Trigger Disable DNSSEC Modal</a
+    >
   {% endif %}
+  {# Use data-force-action to take esc out of the equation and pass cancel_button_resets_ds_form to effectuate a reset in the view #}
+  <div
+    class="usa-modal"
+    id="toggle-dnssec-alert"
+    aria-labelledby="Are you sure you want to continue?"
+    aria-describedby="Your DNSSEC records will be deleted from the registry."
+    data-force-action
+  >
+      {% include 'includes/modal.html' with cancel_button_resets_ds_form=True modal_heading="Warning: You are about to delete all DS records on your domain" modal_description="To fully disable DNSSEC: In addition to deleting your DS records here you’ll also need to delete the DS records at your DNS host. To avoid causing your domain to appear offline you should wait to delete your DS records at your DNS host until the Time to Live (TTL) expires. This is often less than 24 hours, but confirm with your provider." modal_button=modal_button|safe %}
```

Comment:
Please change to:

> **Warning: You are about to disable DNSSEC on your domain**
> Removing all DS data from your domain will disable DNSSEC. To avoid causing your domain to appear offline you should wait at least 24 hours after removing your DS data before disabling DNSSEC at your DNS Host.
> [Delete all DS Data] [Cancel]